doc/nrf/device_guides/pmic/npm1300.rst

```diff
@@ -0,0 +1,132 @@
+
+.. _ug_npm1300_features:
+
+nPM1300 features
+****************
+
+nPM1300 is a PMIC with a linear-mode lithium-ion, lithium-polymer, and lithium ferro-phosphate battery charger.
+It has two dual-mode buck regulators, two dual purpose LDO/load switches, three LED drivers, and five GPIOs.
+
+nPM1300 is the perfect companion for nRF52, nRF53, and nRF54 Series SoCs, and the nRF91 series SiP.
+It is ideal for compact and advanced IoT products such as wearables and portable medical applications.
+
+For additional information on the nPM1300 PMIC and associated kits, see the `nPM1300`_ technical documentation, especially the `nPM1300 EK User Guide`_.
+
+.. _ug_npm1300_compatible_boards:
+
+Boards compatible with nPM1300
+******************************
+
+The following boards in the `Zephyr`_ open source project and in the |NCS| are compatible with nPM1300 PMIC through the nPM1300 EK:
+
+.. list-table::
+   :header-rows: 1
+
+   * - DK or Prototype platform
+     - Companion module
+     - PCA number
+     - Board target
+     - Documentation
+   * - :ref:`zephyr:nrf9160dk_nrf9160`
+     - nPM1300 EK
+     - PCA10090
+     - ``nrf9160dk/nrf9160/ns``
+     - | `Product Specification <nRF9160 Product Specification_>`_
+       | :ref:`Getting started <ug_nrf9160_gs>`
+       | `User Guide <nRF9160 DK Hardware_>`_
+   * - :ref:`zephyr:nrf54h20dk_nrf54h20`
+     - nPM1300 EK
+     - PCA10175
+     - | ``nrf54h20dk_nrf54h20_cpuapp``
+       | ``nrf54h20dk_nrf54h20_cpurad``
+       | ``nrf54h20dk_nrf54h20_cpuppr``
+     - | `Objective Product Specification <nRF54H20 Objective Product Specification 0.3.1_>`_
+       | :ref:`Getting started <ug_nrf54h20_gs>`
+   * - :ref:`zephyr:nrf54l15pdk_nrf54l15`
+     - nPM1300 EK
+     - PCA10156
+     - | ``nrf54l15pdk/nrf54l15/cpuapp`` for the PDK revision v0.2.1, AB0-ES7 (Engineering A).
+       | ``nrf54l15pdk@0.3.0/nrf54l15/cpuapp`` for the PDK revisions v0.3.0 and v0.7.0 (Engineering A).
+     - --
+   * - :ref:`zephyr:nrf5340dk_nrf5340`
+     - nPM1300 EK
+     - PCA10095
+     - ``nrf5340dk/nrf5340/cpuapp``
+     - | `Product Specification <nRF5340 Product Specification_>`_
+       | :ref:`Getting started <ug_nrf5340_gs>`
+       | `User Guide <nRF5340 DK User Guide_>`_
+   * - :ref:`zephyr:nrf52840dk_nrf52840`
+     - nPM1300 EK
+     - PCA10056
+     - ``nrf52840dk/nrf52840``
+     - | `Product Specification <nRF52840 Product Specification_>`_
+       | :ref:`Getting started <ug_nrf52_gs>`
+       | `User Guide <nRF52840 DK User Guide_>`_
+   * - :ref:`zephyr:nrf52dk_nrf52832`
+     - nPM1300 EK
+     - PCA10040
+     - ``nrf52dk/nrf52832``
+     - | `Product Specification <nRF52832 Product Specification_>`_
+       | :ref:`Getting started <ug_nrf52_gs>`
+       | `User Guide <nRF52 DK User Guide_>`_
+
+PMIC samples and libraries
```

Comment:
Should the samples and libraries section be moved towards the end of the documentation. I have see this is what we follow for at least for the integration template?

Reply:
In this case, I think it makes sense to keep this section right after the compatible boards, given how building and programming for PMIC works -- you need a sample matching a board.